nixos/server/default.nix

```diff
@@ -11,6 +11,7 @@
   environment.systemPackages = [
     pkgs.curl
     pkgs.dnsutils
+    pkgs.git
```

Comment:
how about using gitMinimal so it doesn't increase the closure size too much?

Reply:
Minimal would work but sometimes would want pkgs.git in systemPackages. I've set lowPrio for all of these packages, guess it's the closest to mkDefault?